README.md

```diff
@@ -99,6 +99,50 @@ You will need to provide your own CSS styles for hints.
 
 Please see the [instructions on how to install Foundation in a Rails app](http://foundation.zurb.com/docs/applications.html).
 
+### TailwindCSS 3
+
+To generate wrappers that are compatible with [TailwindCSS 3](https://tailwindcss.com/), pass
+the `tailwindcss` option to the generator, like this:
+
+```console
+rails generate simple_form:install --tailwindcss
+```
+
+Then add to your `tailwind.config.js` following config:
+
+```
+module.exports = {
+  content: [
+    './config/initializers/simple_form_tailwindcss.rb'
+  ]
+}
+```
+
+You have to be sure that you have a [tailwindcss-rails](https://github.com/rails/tailwindcss-rails) gem.
+
+**Warnings**
+
+Please pay attention that Tailwindcss not have owned a file-input style and with it, we can't modify `abbr` (star for required fields), so for styling, just add the next styles to your `app/assets/application.tailwind.css` or if you have a gem [tailwindcss_merger](https://github.com/loqimean/tailwindcss_merger) (for multiple files of tailwind's' additional styles) to `app/assets/tailwindcss_stylesheets/your_styles.css`:
+
+*For file-input:*
+
+```
+input[type=file]::-webkit-file-upload-button,
+input[type=file]::file-selector-button {
+   @apply text-white bg-indigo-600 hover:bg-indigo-700 font-medium text-sm cursor-pointer border-0 py-2.5 pl-8 pr-4;
+   margin-inline-start: -1rem;
+   margin-inline-end: 1rem;
+}
+```
+
+*For abbr of required fields:*
+
+```
+abbr[title=required] {
+  @apply !text-red-500 no-underline;
+}
```

Comment:
Hi @loqimean! Thanks for your contribution!

I noticed that we can use the following snippet at the `config/locales/simple_form.en.yml` file and override the `abbr` props

```md
    required:
      # text: "required"
      # mark: "*"
      html: '<abbr class="text-red-500 no-underline" title="required">*</abbr>'
```

![image_2023-09-03_002728140](https://github.com/heartcombo/simple_form/assets/7331511/6d9e7912-b9dc-43ba-b249-06a2878f0e63)

Reply:
oh, nice idea, can you add it to the PR?)

Reply:
Sure will add it.Actually, I'm creating a gem called `simple_form_themer` (open to change the name if any suggestion) to support more CSS themes on it. I had worked on bulma + simple_form so I have the base of it.

Reply:
@loqimean I ended up creating a gem to install the tailwind theme and added the locale file automatically. (It also has support for bulma css)

https://github.com/JuanVqz/simple_form-theme/pull/5